locales/en/messages.json

```diff
@@ -3881,6 +3881,9 @@
     "osdDescElementLogStatus": {
         "message": "Blackbox number and warnings"
     },
+    "osdDescElementOsdProfileName": {
+        "message": "OSD profile name as set in the \"osd_profile_1_name\", \"osd_profile_2_name\" and \"osd_profile_3_name\" cli variables"
```

Comment:
I don't know if it is a good idea to reference the CLI variables. At this stage is ok, but later, in future versions, it will be added to the Configurator and we must remember change them here. What do you think @mikeller? Is ok to add them here?

Reply:
While I agree that this should be changed once the names are accessible in the UI, I think that proactively giving users this information now is will help reduce the likelihood of issues being opened about this, so it should be done.

I'd probably capitalise 'CLI' though.